Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -971,7 +971,7 @@ extension ProjectDescription.TargetDependency {
                     .linker,
                     .define
                 ),
-                (.linker, .unsafeFlags), (_, .enableExperimentalFeature):
+                (.linker, .unsafeFlags), (_, .enableExperimentalFeature), (_, .swiftVersion):
```

Comment:
I don't fully understand this linker dependency stuff so this should be reviewed by someone.

Reply:
Same as above, this is fine